ci/mill-bootstrap.patch

```diff
@@ -1,3 +1,18 @@
+diff --git a/build.mill b/build.mill
+index d815508132a..ba62e147383 100644
+--- a/build.mill
++++ b/build.mill
+@@ -19,9 +19,8 @@ import mill.define.Cross
+ import scala.util.Properties
+ 
+ // plugins and dependencies
+-import $meta._
++//| meta: true
+ 
+-import $packages._
```

Comment:
Did you remove the need for `packages` enablement? Also I find just `meta` a bit short and undescriptive. What about `meta-build: true` or `mill.features: [meta,packages]`.

Reply:
Maybe it just about more automated assistance that's needed. I could think of in case we hit an error, we check if we can find those files and ask the user if she meant to enable those features. Or we always check if the feature is not disabled, and warn the user that we are going to ignore those files.

Reply:
Maven and Gradle have the explicit listing of subpackages similar to what you propose here, e.g. Maven's `<modules>` config or Gradle's `include` statements, so there is precedence. Though I must say I've hit the same confusion working with Maven/Gradle builds where I move stuff around the filesystem and forget to update the `<modules>` or `include` statements, or vice versa